R4ML/R/ml.preprocess.R

```diff
@@ -596,3 +596,32 @@ isAnyAttrOfType <- function(hm, attrs, type) {
   return(any(allColumnTypes[allColumnNames %in% attrs] == type))
 }
 
+# Method to transform a r4ml.frame to a r4ml.matrix that can be read by DML 
+#'
+#' @name r4ml.systemml.transform
+#' @title Wrapper method for \code{\link{r4ml.ml.preprocess}}
+#' @description Wrapper method for \code{\link{r4ml.ml.preprocess}} and produces a \code{r4ml.matrix}.
```

Comment:
If output is non numeric, then ideally in future as.r4ml.matrix will error out in the L626. But for now it won't so we should have this message that it is recommended to use ml.preprocess. Only difference between this and other is that this is wrapper which just coerce the output to be matrix